site/docs/how-tos/operator-openshift.md

```diff
@@ -0,0 +1,298 @@
+---
+title: How to install Hazelcast Jet Enteprise Operator on Red Hat
```

Comment:
should this be on enterprise section maybe?

Reply:
Your call, but it won't be discoverable from the main sidebar, wdyt ?

Reply:
yes that's fine, it will be in enterprise edition sidebar. Title is also too long for the sidebar